### ISSUE TITLE
Change the repository type to a "template repository"

### ISSUE THREAD

**Issue by luk27official** [exaexa/better-mff-thesis#8]
I think that the type of this repository could be changed to a GitHub's "template repository".

This allows usage of this template without forking the entire repository including all past commits, does not show the fork's commits in the original repo's contribution graph etc. More information about GitHub's template repos including all differences may be found [here](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-template-repository) and [here](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-repository-from-a-template).

In short, repositories derived from the "templates" are more independent of the original repo. I think that the GitHub's description pretty much satisfies the goals of this repository.

Thanks in advance for consideration of this change.

**Comment by exaexa** [comment]
Hello!

there's actually a slightly less-unofficial fork of this repo here https://github.com/mff-cuni-cz/better-thesis/ which has the "use the template" button. But I guess there's no reason for not marking this templatable too, good point, done now. Thanks for the notice.

Re forking vs. templating, I'm slightly inclined towards "sharing is caring" and having a network of forks with a diverse sample of common thesis problems solved is a seriously helpful resource, but we can't push anyone to that :D

Best!
-mk
